rollup.config.mjs

```diff
@@ -39,8 +39,7 @@ export default [
 	{
 		input: {
 			...runtime_entrypoints,
-			index: 'src/runtime/index.ts',
-			ssr: 'src/runtime/ssr.ts'
+			index: 'src/runtime/index.ts'
```

Comment:
why was this removed? This effectively stops the `ssr` output from being generated, so the exports map points at a nonexistent file.

Reply:
yeah, you're absolutely right. I was thinking at one point it'd be unused, but that's clearly not the case. reverted that. hopefully this PR is good to go now